### ISSUE TITLE
[Suggestion] Create a flag to contain folders

### ISSUE THREAD

**Issue by jopemachine** [ytakahashi/chrome-bookmark-reader#11]
### Suggestion

Thanks for creating great lib :)

How about creating an option to contain the `folder` type in the return value if some flag is on?

(I think there might be someone who wants to receive folder type only maybe..)

I think it can be implemented as simply as below.

Example code:
```ts
// chromeBookmark.ts

...
  getAsArray(): Array<BookmarkElement> {
    if (this.hasChildren() && someFlagOn) {
      // contain 'this' (folder type)
      return [...this.children.flatMap(c => c.getAsArray()), this]
      // contain only url type
    } else if (this.hasChildren()) {
      return this.children.flatMap(c => c.getAsArray())
    }

    return [this]
  }

```

**Comment by ytakahashi** [comment]
Hi @jopemachine , thank you for the suggestion and example!
Yes, I think it's possible to include folders. I will try to do soon.

**Comment by ytakahashi** [comment]
Hi, I did in #12 and published 0.4.0. Thank you!
